plugins/modules/docker_container.py

```diff
@@ -919,6 +920,8 @@
         with the requested config.'
       - 'V(started) - Asserts that the container is first V(present), and then if the container is not running moves it to a running
         state. Use O(restart) to force a matching container to be stopped and restarted.'
+      - V(healthy) - Asserts that the container is V(present) and V(started), and is actually healthy as well.
```

Comment:
```suggestion
      - V(healthy) - Asserts that the container is V(present) and V(started), and is running without errors.
```

Reply:
Not sure if that's the most accurate description of the healthy state but it avoids using the word "healthy" to describe the state "healthy".

Reply:
Hmm, I'm a bit hesitant about this, since `healthy` is the terminology for Docker containers (https://docs.docker.com/reference/dockerfile/#healthcheck).

How about the following:
```suggestion
      - V(healthy) - Asserts that the container is V(present) and V(started), and is actually healthy as well.
        This means that the conditions defined in O(healthcheck) respectively in the image's C(HEALTHCHECK)
        (L(Docker reference for HEALTHCHECK, https://docs.docker.com/reference/dockerfile/#healthcheck))
        are satisfied.
```

Reply:
@felixfontein Ah, I see. I think that suggestion is a good improvement for sure.